### ISSUE TITLE
PureeBufferedOutput not work with SamplingFilter

### ISSUE THREAD

**Issue by yss44** [cookpad/puree-android#36]
PureeBufferedOutput#insertSync may be like below?

```
    @Override
    public void insertSync(String type, JsonObject jsonLog) {
        JsonObject filteredLog = applyFilters(jsonLog);
        if(filteredLog == null) {
            return;
        }
        storage.insert(type, filteredLog);
    }

```

**Comment by gfx** [comment]
This is fixed in v3.3.0. Thanks.

**Comment by yss44** [comment]
:smiley:
